corehq/apps/app_manager/static/app_manager/js/details/case_claim.js

```diff
@@ -268,47 +268,25 @@ hqDefine("app_manager/js/details/case_claim", function () {
-                search_label: self.searchLabel(),
-                search_label_image:
-                    $("#case_search-search_label_media_media_image input[type=hidden][name='case_search-search_label_media_media_image']").val() || null,
-                search_label_image_for_all:
-                    $("#case_search-search_label_media_media_image input[type=hidden][name='case_search-search_label_media_use_default_image_for_all']").val() || null,
-                search_label_audio:
-                    $("#case_search-search_label_media_media_audio input[type=hidden][name='case_search-search_label_media_media_audio']").val() || null,
-                search_label_audio_for_all:
-                    $("#case_search-search_label_media_media_audio input[type=hidden][name='case_search-search_label_media_use_default_audio_for_all']").val() || null,
-                search_again_label: self.searchAgainLabel(),
-                search_again_label_image:
-                    $("#case_search-search_again_label_media_media_image input[type=hidden][name='case_search-search_again_label_media_media_image']").val() || null,
-                search_again_label_image_for_all:
-                    $("#case_search-search_again_label_media_media_image input[type=hidden][name='case_search-search_again_label_media_use_default_image_for_all']").val() || null,
-                search_again_label_audio:
-                    $("#case_search-search_again_label_media_media_audio input[type=hidden][name='case_search-search_again_label_media_media_audio']").val() || null,
-                search_again_label_audio_for_all:
-                    $("#case_search-search_again_label_media_media_audio input[type=hidden][name='case_search-search_again_label_media_use_default_audio_for_all']").val() || null,
-                search_filter: self.searchFilter(),
-                blacklisted_owner_ids_expression: self.blacklistedOwnerIdsExpression(),
-                additional_registry_cases: self.dataRegistryWorkflow() === "load_case" ?  self.additionalRegistryCases().map((query) => {
-                    return query.caseIdXpath();
-                }) : [],
-                custom_related_case_property: self.customRelatedCaseProperty(),
-                inline_search: self.inlineSearch(),
-            };
+            var data = ko.mapping.toJS(self);
+            data.additional_registry_cases = data.data_registry_workflow === "load_case" ?  _.pluck(data.additional_registry_cases, 'caseIdXpath') : [];
+            _.each(['search_label', 'search_again_label'], function (label) {
+                _.each(['image', 'audio'], function (media) {
+                    var key = label + "_" + media,
+                        selector = "#case_search-" + label + "_media_media_" + media + " input[type='hidden']";
+                    data[key] = $(selector + "[name='case_search-" + label + "_media_media_" + media + "']").val();
```

Comment:
The removal of all the `null` fallbacks in the old code is intentional. These values get [passed to set_media](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/views/modules.py#L1223-L1224), which [defaults them to blank string anyway](https://github.com/dimagi/commcare-hq/blob/4f03452ae7cf8e86bd66fe52223a466f11c3358f/corehq/apps/app_manager/models.py#L1567).